src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_VALID_LOGIN = "Yaroslav";
+    private static final String DEFAULT_VALID_PASSWORD = "123456";
+    private static final int DEFAULT_VALID_AGE = 18;
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_loginNull_NotOk() {
+        User user = new User(null, DEFAULT_VALID_PASSWORD, DEFAULT_VALID_AGE);
```

Comment:
I think you can @BeforeEach create a default user using your constants and then change the value of that user using setters if necessary.

Reply:
in my opinion, this option is more informative